Sources/Components/Fiks Ferdig/FiksFerdigInfoView/FiksFerdigShippingInfoCell.swift

```diff
@@ -0,0 +1,92 @@
+import UIKit
+import FinniversKit
+
+public final class FiksFerdigShippingInfoCell: UIStackView {
+    private let viewModel: FiksFerdigShippingInfoCellViewModel
+
+    private let providerLabel: Label = {
+        let label = Label(style: .captionStrong, withAutoLayout: true)
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private let messageLabel: Label = {
+        let label = Label(style: .caption, withAutoLayout: true)
+        label.numberOfLines = 0
+        return label
+    }()
```

Comment:
`numberOfLines` has been added as an init parameter, so these two can be shortened:

```swift
private let providerLabel = Label(style: .captionStrong, numberOfLines: 0, withAutoLayout: true)
private let messageLabel = Label(style: .caption, numberOfLines: 0, withAutoLayout: true)
```

Reply:
@bstien It doesn't show up in the latest version of the Finniverse SPM after I tried updating it. 
<img width="211" alt="image" src="https://user-images.githubusercontent.com/11469405/208851235-e9f51cb4-de04-4226-9d80-67e7f4ab4a8a.png">

Should I change that SPM reference to point to `master` instead?

Reply:
No, don't use master unless in emergency. :D Check if FinniversKit is ready for a version bump and make a new release if that is what's required.